src/__fixtures__/authUsers.ts

```diff
@@ -1,5 +1,23 @@
 import { ObjectId } from 'bson'
-import type { SessionUser, PortalUser } from 'types'
+import type { SessionUser, PortalUser, Collection } from 'types'
+
+const mockNews: any = {
+  _id: new ObjectId(),
+  title: 'Recent News',
+  type: 'News',
+}
```

Comment:
I don't like having to use `any`, but I tried to get the type right ten different ways and it just wasn't working for this one, as well as the few other times I had to use `any` in this file. The issue lies in the `Collection` interface extending the `Widget` type. The `bookmarks` field is not on `Widget` but is required by `Collection`. I figured `any` would be fine for now since this is mock data.